hazelcast/src/main/resources/hazelcast-config-5.1.json

```diff
@@ -1497,7 +1497,7 @@
                 "additionalProperties": false,
                 "properties": {
                   "capacity": {
-                    "$ref": "#/definitions/MemorySize",
+                    "$ref": "#/definitions/Capacity",
```

Comment:
This is `hazelcast-config-5.1.json`, is this change a mistake?

Reply:
Yeah, changed should be in 5.2 only. 316de23b436bfd2578cf7ef48ea185be80dcfe23